enclave_mgmt/model_creation.py

```diff
@@ -0,0 +1,414 @@
+import pandas as pd
```

Comment:
Similar suggestion as far as alternate name: What about calling this `create_models.py`?